### PR TITLE
Fast Stop

### PR DESCRIPTION
**Pull request by SunzBody**
https://www.youtube.com/watch?v=3QihqT1GkH8

Implement in Osiris thanks to @notgoodusename

### CODE REVIEW

**Comment by sserve-kr** [comment]
where is write command? (in config.cpp)

**Comment by SunzBody** [comment]
Thanks, I'll take care of it, I'd done that without even checking if the DLL was working properly

**Comment by SunzBody** [comment]
Do you think this feature could be added to the original Osiris repo ? @danielkrupinski

**Comment by dayz2000** [comment]
Conflicting files
Osiris/Hooks.cpp

can u fix ?

**Comment by dayz2000** [comment]
> Conflicting files
> Osiris/Hooks.cpp
> 
> can u fix ?

ok manually add code 
still ok

File: Osiris/Hacks/Misc.cpp [hunk 1]
```diff
@@ -291,6 +291,75 @@ void Misc::fastPlant(UserCmd* cmd) noexcept
...
+            if (angles.y > 90)
+                angles.y -= 180;
+            else if (angles.y < 90)
+                angles.y += 180;
+            else if (angles.y == 90)
+                angles.y = 0;
+        }
+
+        angles.z = 0.0f;
+    };
```
Comment (danielkrupinski) [inline Osiris/Hacks/Misc.cpp:329]: Use `Vector::toAngle()`

Comment (danielkrupinski) [inline Osiris/Hacks/Misc.cpp:329]: Use `Vector::toAngle()`

File: Osiris/Hacks/Misc.cpp [hunk 2]
```diff
@@ -291,6 +291,75 @@ void Misc::fastPlant(UserCmd* cmd) noexcept
...
+        sy = sin(degreesToRadians(angles.y));
+        cy = cos(degreesToRadians(angles.y));
+
+        sp = sin(degreesToRadians(angles.x));
+        cp = cos(degreesToRadians(angles.x));
+
+        forward->x = cp * cy;
+        forward->y = cp * sy;
+        forward->z = -sp;
+    };
```
Comment (danielkrupinski) [inline Osiris/Hacks/Misc.cpp:343]: Use `Vector::fromAngle()`

**Review by danielkrupinski — APPROVED** [review]

**Comment by danielkrupinski** [comment]
Thanks!

**Pull request merged**